lib/backpex/resource.ex

```diff
@@ -271,7 +271,7 @@ defmodule Backpex.Resource do
     |> maybe_preload(associations, fields)
     |> maybe_merge_dynamic_fields(fields)
     |> where([{^schema_name, schema_name}], schema_name.id == ^id)
-    |> repo.one()
+    |> repo.one!()
```

Comment:
@Flo0807 This will raise a `Ecto.NoResultsError` and show a proper 404 page instead of returning `nil` and failing later on. Should we rename `get` to `get!` or `one!`?

Reply:
> Should we rename get to get! or one!?

yes, I like`get!`